src/array/array.cc

```diff
@@ -810,7 +812,7 @@ COOMatrix COORowWiseSampling(
     if (IsNullArray(prob)) {
       ret = impl::COORowWiseSamplingUniform<XPU, IdType>(mat, rows, num_samples, replace);
     } else {
-      ATEN_FLOAT_TYPE_SWITCH(prob->dtype, FloatType, "probability", {
+      ATEN_FLOAT_INT8_UINT8_TYPE_SWITCH(prob->dtype, FloatType, "probability", {
```

Comment:
do we need to change the `val_name`: `probability` to indicate `prob=prob or mask`?